app/assets/stylesheets/home.scss

```diff
@@ -3,14 +3,13 @@
   margin-bottom: 1rem;
 }
 .about {
-  padding: 2rem;
+  padding: 1rem;
   background-color: #d5ffea;
-  border-radius: 50px;
 }
 
 .example-chart {
-  background-color: #d5ffea;
-  border-radius: 50px;
+  background-color: #F3F3F3;
```

Comment:
Color literals like `#F3F3F3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F3F3F3` should be written as `#f3f3f3`